tests/mason-core/installer/managers/std_spec.lua

```diff
@@ -1,8 +1,16 @@
 local installer = require "mason-core.installer"
 local match = require "luassert.match"
+local platform = require "mason-core.platform"
 local std = require "mason-core.installer.managers.std"
 local stub = require "luassert.stub"
 
+local function get_tar_cmd(ctx)
+    if platform.is.unix and not platform.is.linux then
+        return ctx.spawn.gtar
+    end
+    return ctx.spawn.tar
+end
+
```

Comment:
By leveraging `vim.fn.executable("gtar")` we can keep these tests the same they were before without having to incorporate any logic.

```lua
describe("...", function ()
  before_each(function ()
    stub(vim.fn, "executable")
    vim.fn.executable.on_call_with("gtar").returns(0)
  end)
end)
```

Then we can also add a dedicated test for gtar detection:

```lua
it("should use gtar if available", function ()
  -- … boilerplate
  stub(vim.fn, "executable")
  vim.fn.executable.on_call_with("gtar").returns(1)
  -- … execute & assert
end)
```